### ISSUE TITLE
The Split of R2D road dataset

### ISSUE THREAD

**Issue by csBob123** [hlwang1124/SNE-RoadSeg#2]
Hi, 

Thank you for your work. Could I know how to split the R2D road dataset?

 Which part belongs to training and which part belongs to the evaluation(The split files)?
 
Many thanks

**Comment by hlwang1124** [comment]
In our experiments, we follow the following dataset split scheme (T is short Town):
training: T2, T3, T4 and T5 sunset wet;
validation: T1 and T5 sunset clear;
testing: T5 day wet and T6.

**Comment by csBob123** [comment]
Thank you for your reply.

Thus, to get the result in Table 2, first, we need to train the model on the training set. Then, using the validation set to select out the best model. Finally,  the best model will be used to get the results on the testing set and the results will be reported in Table 2.

Is this right?

So sorry for my troubles.

Thanks

**Comment by hlwang1124** [comment]
Yes. If everything works fine, you will get similar results as reported in the paper.
